drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -172,12 +172,12 @@ void at86rf2xx_set_page(at86rf2xx_t *dev, uint8_t page)
 
 uint16_t at86rf2xx_get_pan(at86rf2xx_t *dev)
 {
-    return dev->pan;
+    return byteorder_ntohs(dev->pan);
 }
 
 void at86rf2xx_set_pan(at86rf2xx_t *dev, uint16_t pan)
 {
-    dev->pan = pan;
+    dev->pan = byteorder_htons(pan);
```

Comment:
Fixed.

@gebart: I know! Again: my comment wasn't about the order in the MAC frame (this was and still is right), but in the RIOT internal representation and storing according [to the documentation](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/netopt.h#L55).